src/schema/messaging/received.ts

```diff
@@ -15,7 +16,7 @@ export default defineEndpoint({
                     player_id: Type.String(),
```

Comment:
We use `src/schema/definitions/userId.ts` everywhere else, maybe it should be dropped and just a string, but at least the name of property would be nice to name `userId`. The same below.